<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>588</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Listado de Responsables</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>601</width>
     <height>361</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="dragEnabled">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::SingleSelection</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <column>
    <property name="text">
     <string>CI</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Edad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sexo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rol</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ocupación</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btn_insertar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Insertar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_modificar">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Actualizar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_eliminar">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Eliminar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_nombre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ci">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>CI</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_edad">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Edad</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_rol">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>60</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Departamento</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nombre">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_ci">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_rol">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>60</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Local</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Area</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spin_edad">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>120</number>
   </property>
   <property name="value">
    <number>18</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Sexo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ocupación</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_sexo">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Masculino</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Femenino</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_exportar">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Importar</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_ocupacion">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>100</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_buscar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
